aws/basic-eks-cluster/kubernetes/aws-auth-cm.yaml

```diff
@@ -10,8 +10,10 @@ data:
       groups:
         - system:bootstrappers
         - system:nodes
+  {{if .AddIAMUser }}
   mapUsers: |
-    - userarn: ##EksUserArn##
-      username: ##EksUser##
+    - userarn: ##EKS_USER_ARN##
```

Comment:
These variable names are incorrect. Actually previous CamelCase version was right. In dictionary u have `EksUser: '{{"{{%outputVariables.EksUser%}}"}}'
EksUserArn: '{{"{{%outputVariables.EksUserArn%}}"}}`

Reply:
fixed